components/Chat/ChatMarkdown.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Markdown from "react-markdown";
+import remarkGfm from 'remark-gfm';
+import styles from "./markdown.module.css";
+import MarkdownA from "./MarkdownA";
+import MarkdownPre from "./MarkdownPre";
+import MarkdownCode from "./MarkdownCode";
+import MarkdownImg from "./MarkdownImg";
+import MarkdownTable from "./MarkdownTable";
+
+interface ChatMarkdownProps {
+  children: string;
+}
+
+const ChatMarkdown: React.FC<ChatMarkdownProps> = ({ children }) => {
+  return (
+    <div className={`${styles.markdown} w-full max-w-full overflow-hidden`}>
+      <Markdown
+        remarkPlugins={[remarkGfm]}
+        components={{
+          a: ({ ...props }) => <MarkdownA {...props} />,
```

Comment:
Would this be the same thing?
```suggestion
          a: MarkdownA,
```